camel-openmrs/src/main/java/org/openmrs/eip/component/Constants.java

```diff
@@ -40,6 +40,8 @@ public class Constants {
 	
 	public static final List<String> SUBCLASS_TABLES;
 	
+	public static final String EVENT_DATE = "eventDate";
```

Comment:
You don't need this constant here, it should be defined at the subclass level of the controller

Reply:
Done